### ISSUE TITLE
allow to pass command line switchs to dmenu

### ISSUE THREAD

**Issue by fdietze** [gustavnikolaj/keepass-dmenu#9]

**Comment by gustavnikolaj** [comment]
How do you want the api for this to be? Any ideas? :-)

**Comment by fdietze** [comment]
I can think of directly accepting all dmenu switches  (or only the ones which make sense) and handing them over do demenu.

This would look like:
```
keepass-dmenu --database db.kdbx -nb "#FFFFFF" -nf "#000000" -sb "#00D7FF" -sf "#101010"
```

**Comment by gustavnikolaj** [comment]
You don't want to be able to style the password prompt and password filtering differently?

If the answer to that is no, then this should be straight forward enough. I'll just pass on options, that I don't understand.

**Comment by fdietze** [comment]
One styling for everything is enough. Your suggestion sounds great, thank you!

**Comment by fdietze** [comment]
Now I realize how you hide the password.
Maybe you can make an exception for the password prompt and give the foreground-color the same color as the backgorund-color. The bg color is either handed in or set as default, as you do right now.

**Comment by gustavnikolaj** [comment]
@fdietze Yep, I had to special case the password hiding :-) https://github.com/gustavnikolaj/keepass-dmenu/blob/58d5b6d2223baee73a0c1596d987d3d9e9fbb089/lib/passwordPrompt.js#L7

It's a bit of a hack, but I couldn't come up with anything better.

I opted only to expose fn (the font option) and nb, nf, sb and sf (the color options) - so this is only the styling related ones. We could aswell just pass the rest along - honestly I don't see any reason why. But I'll be lazy and wait until someone needs it ;-)

I hope this works for you :-)
